packages/perseus/src/widgets/orderer/orderer.tsx

```diff
@@ -371,7 +369,7 @@ class Orderer
     ) => {
         // @ts-expect-error - TS2769 - No overload matches this call.
         const $draggable = $(ReactDOM.findDOMNode(draggable));
```

Comment:
Ohhh another findDomNode to get rid of. (More of a self note to help me remember)